performance/pom.xml

```diff
@@ -107,6 +116,29 @@
             </properties>
         </profile>
         <profile>
+            <id>hv-current</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.hibernate</groupId>
+                    <artifactId>hibernate-validator</artifactId>
+                </dependency>
+                <dependency>
+                    <groupId>org.glassfish</groupId>
+                    <artifactId>javax.el</artifactId>
+                </dependency>
+                <dependency>
```

Comment:
Could you add a log4j config which only emits WARNING or ERROR? Currently, I see lots of DEBUG messages on the console which for sure will take an effect on the benchmark.